delivery_carrier_general_manifest/wizards/manifest_wizard.py

```diff
@@ -0,0 +1,91 @@
+# Copyright 2025 Alberto Martínez <alberto.martinez@sygel.es>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+import base64
+
+from odoo import _, api, exceptions, fields, models
+
+
+class ManifestWizard(models.TransientModel):
+    _inherit = "manifest.wizard"
+
+    carrier_id = fields.Many2one(
+        required=False,
+    )
+    general_carrier_ids = fields.Many2many(
+        string="General Carriers",
+        comodel_name="delivery.carrier",
+        domain=[("include_in_general_manifest", "=", True)],
+        compute="_compute_general_carrier_ids",
+        store=True,
+        readonly=False,
+    )
+    general_manifest_query = fields.Boolean(
+        string="General Manifest",
+        help="A general manifest query generates a generic general manifest "
+        "report using all carriers configured to be included on it",
+    )
+    company_id = fields.Many2one(
+        comodel_name="res.company", default=lambda self: self.env.company
+    )
+
+    @api.depends("company_id")
+    def _compute_general_carrier_ids(self):
+        for rec in self:
+            rec.general_carrier_ids = (
+                self.env["delivery.carrier"]
+                .search(
+                    [
+                        ("include_in_general_manifest", "=", True),
+                        ("company_id", "in", [False, rec.company_id.id]),
+                    ]
+                )
+                .ids
+            )
+
+    def _get_general_manifest_pickings(self):
+        self.ensure_one()
+        search_args = [
```

Comment:
@ValentinVinagre @Tisho99 My only concern is about this to retrieve the correct pickings.

Have you ever evaluated this : https://github.com/OCA/stock-logistics-transport/tree/16.0/shipment_advice

As it allows to load outgoing pickings in a shipment advice object. That allows you to be sure the pickings you try to print manifest are really the good ones.

Reply:
Hmm, I think they're different functionalities. The idea is that at the end of the day, or at some point during the day, the person in charge can create a report/printout of everything that has been delivered to carriers and to which ones.
I think the "shipment_advice" module is more complex.